app/components/NewsSection.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Collection,
+  CollectionCalendarDate,
+  CollectionDescription,
+  CollectionHeading,
+  CollectionItem,
+  Link,
+} from '@trussworks/react-uswds'
+import type { ReactNode } from 'react'
+
+export interface NewsProps {
+  Date: string
+  Title: string
+  BodyText: ReactNode
+}
+
+export default function NewsSection({ news }: { news?: NewsProps[] }) {
+  const newsList: NewsProps[] = [
+    {
```

Comment:
I don't think that this saves any space as compared to having a literal fragment of `<CollectionItem />` components in `news.mdx`.

Reply:
This is just because of how it was implemented on the other branch I had been working on. But I can just merge them into the one file if you think it is better that way